File: Java/electricaloutlets.java
/*
 *	Problem: "Electrical Outlets"
 */

import java.util.Scanner;

public class electricaloutlets {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int cases = sc.nextInt();
		
		for (int i = 0; i < cases; i++) {
			int maxPowered = 0;
			int numStrips = sc.nextInt();

			for (int j = 0; j < numStrips; j++) {
				int numOutlets = sc.nextInt();
				maxPowered += numOutlets;
			}
			int occupiedOutlets = numStrips - 1;
			maxPowered -= occupiedOutlets;			
			System.out.println(maxPowered); 
		}
		
	}
}

File: Python/electricaloutlets.py
'''
 *
 *	Problem: "Electrical Outlets"
 *
'''

cases = int(input())

for case in range(cases):
	maxPowered = 0
	
	line = input().split(" ")
	numStrips = int(line[0])
	
	for strip in range(1, (numStrips + 1)):
		numOutlets = int(line[strip])
		maxPowered += numOutlets

	occupiedOutlets = numStrips - 1
	maxPowered -= occupiedOutlets
	
	print(maxPowered)